tidy3d/web/api/tidy3d_stub.py

```diff
@@ -0,0 +1,87 @@
+"""Stub for webapi"""
+from __future__ import annotations
+
+from typing import Union
+
+from pydantic.v1 import BaseModel
+
+from tidy3d import SimulationData, ModeSolverData
+from tidy3d.web.core.types import TaskType
+from tidy3d.components.simulation import Simulation
+from tidy3d.plugins.mode.mode_solver import ModeSolver
+
+
+class Tidy3dStub(BaseModel):
```

Comment:
What happened to the `Stub` and `StubData`? I thought we were going to define an interface to define what the tidy3d side objects needed to implement to work with core?

Reply:
added